[RHACS] Added a note about Amazon S3 API-compatible storage (#76274)
openshift/openshift-docs: backup_and_restore/backing-up-acs.adoc

You can use the `roxctl` CLI to take the backups by using the `backup` command. You require an API token or your administrator password to run this command.

[NOTE]
====

> Red Hat supports backups for the Central database by integrating with xref:../integration/integrate-with-amazon-s3.adoc#perform-on-demand-backups-amazon-s3_integrate-with-amazon-s3[Amazon S3] or xref:..//integration/integrate-with-google-cloud-storage.adoc#perform-on-demand-backups-google-cloud-storage_integrate-with-google-cloud-storage[Google Cloud Storage]. Backing up on Amazon S3 API-compatible storage might work, however, Red Hat does not test and support Amazon S3 API-compatible storage for backing up {product-title-short}. For example, see link:https://access.redhat.com/solutions/7021082[{product-title-short} backup on MinIO].

Reviewer: ```suggestion
Red Hat supports backups for the Central database through integration with xref:../integration/integrate-with-amazon-s3.adoc#perform-on-demand-backups-amazon-s3_integrate-with-amazon-s3[Amazon S3] or xref:../integration/integrate-with-google-cloud-storage.adoc#perform-on-demand-backups-google-cloud-storage_integrate-with-google-cloud-storage[Google Cloud Storage]. Backing up to Amazon S3 API-compatible storage might work, however Red Hat does not test or support Amazon S3 API-compatible storage for backing up {product-title-short}. 

For example, see link:https://access.redhat.com/solutions/7021082[{product-title-short} backup on MinIO].
```

Reviewer: "xref:..//"
I have never seen this usage in our docs. I seem to recall @kcarmichael08  highlighting this usage in the peer review channel in one of the PRs she reviewed. I see that it does not interfere with the preview right now, but we are not sure if it can cause issues in the future. In our case here, we are just one level deep, so I guess xref:../ would be sufficicent. Just thinking aloud here!

Author: Thank you @agantony That's a mistake, it should be one `/`. 🙇🏼

Author: I'm still using `test and support` because using `test or support` can mean that we might do one or the other.
